Update README.md (#4)
viam-labs/aws-sagemaker: README.md

# aws-sagemaker

> A module providing a model of the Viam vision service that allows you to access vision ML models deployed on cloud endpoints using AWS Sagemaker.

Author: @kharijarrett Done!

> Module of the Viam vision service to access models deployed on cloud endpoints using AWS Sagemaker.  This module allows you to access vision ML models that have been deployed to an AWS Sagemaker endpoint.

Author: ```suggestion
A module providing a model of the Viam vision service that allows you to access vision ML models deployed on cloud endpoints using AWS Sagemaker.
```
@kharijarrett Maybe it would make more sense to just combine the two sentences if they're effectively stating the same thing?

Reviewer: Yeah it does but maybe "which" --> "that"
